client/src/components/order/new-order.js

```diff
@@ -73,13 +75,28 @@ class NewOrder extends Component {
     } else {
       return;
     }
+    let orderedDishes = this.props.location.state.menuItemDishesMap;
```

Comment:
Simpler and more readable [{dishId: 25, quantity: 3}, {dishId: 26, quantity: 2}, {dishId: 29, quantity: 2}]
![image](https://user-images.githubusercontent.com/6152066/80941185-713e6680-8dea-11ea-9911-7ba84728e8dd.png)

Reply:
This json is automatically converted from map object. To implement another view we should add some logic to save our Map in List both frontend and backend. This isn't necessary.

Reply:
In this case my comment is only recommendation, you don't need to follow it